### DISCUSSION TITLE
How to search for transactions after a relative number of days ago

### DISCUSSION THREAD

**Discussion by digitlength** [firefly-iii/firefly-iii#8726] [Firefly III] [answered]
Following the documentation at https://docs.firefly-iii.org/references/firefly-iii/search/ I thought I might be able to do:

`date_after:-3d`

...which would give me all the transactions _starting_ from 3 days ago. Unfortunately, the search says that this is invalid syntax.

Is there a way for me to grab all transactions starting from X days ago, where X is a constant, and without having to write the specific date itself?

**Comment by JC5** [comment]
Mmm, I'm not even sure if I added that. I'll check it out and get back to you.

**Comment by JC5** [comment, marked as answer]
Add quotes to your search:

`date_after:"-3d"`
